diesel_cli/src/main.rs

```diff
@@ -396,14 +396,9 @@ fn run_infer_schema(matches: &ArgMatches) -> Result<(), Box<dyn Error>> {
     let filter = matches
         .values_of("table-name")
         .unwrap_or_default()
-        .map(|table_name| {
-            if let Some(schema) = config.schema_name() {
-                TableName::new(table_name, schema)
-            } else {
-                table_name.parse().unwrap()
-            }
-        })
-        .collect();
+        .map(|table_name_regex| Regex::new(table_name_regex).map(Into::into))
+        .collect::<Result<_, _>>()
+        .map_err(|e| format!("invalid argument for table filtering regex: {}", e));
 
     if matches.is_present("whitelist") {
```

Comment:
As already written: `whiteliste`/`blacklist` has been deprecated since [version 1.3.0](https://github.com/diesel-rs/diesel/blob/master/CHANGELOG.md#130---2018-05-22) we should completely remove them from the CLI.

Reply:
Oh, I misunderstood. I'm going to remove these flags.